java_basic_programming/2019-10-22_Anagrams/src/de/telran/Main.java

```diff
@@ -0,0 +1,58 @@
+package de.telran;
+
+import java.util.*;
+
+public class Main {
+
+    public static void main(String[] args) {
+	    List<String> anagrams = new ArrayList<>();
+	    anagrams.add("anna");
+        anagrams.add("nana");
```

Comment:
You may check for various shorter ways of populating a list with values than just by adding values one by one. Example: Arrays.asList() method.